src/store/store.js

```diff
@@ -0,0 +1,85 @@
+import { registration, logIn, logOut, refresh } from 'src/services/AuthService';
+import { getMeetings } from 'src/services/MeetingService';
+
+export default class Store {
```

Comment:
if you correctly put the dependencies of the branches, then the code will not be duplicated